src/topic_partition_list.cpp

```diff
@@ -89,7 +89,7 @@ TopicPartitionList find_matches(const TopicPartitionList& partitions,
         for (const auto& topic : topics) {
             if (topic.size() == partition.get_topic().size()) {
                 // compare both strings
-                bool match = equal(topic.begin(), topic.end(), partition.get_topic().begin(),
+                bool match = std::equal(topic.begin(), topic.end(), partition.get_topic().begin(),
```

Comment:
Rather than this, I'd add a `using std::equal` right below the `include` directives.